onnxruntime/core/mlas/lib/mlasi.h

```diff
@@ -836,6 +885,51 @@ struct MLAS_PLATFORM {
     static constexpr int32_t MaximumThreadCount = MLAS_MAXIMUM_THREAD_COUNT;
 #endif
 
+#if defined(MLAS_TARGET_ARM64) && defined(__linux__)
+    // TODO!! implement uarch detection in Windows
+    std::vector<MlasCoreType> mlas_coretype_tbl;
+#endif
+
+    /**
+     *  @return 2 current core is little core with narrow memory load (e.g. ARMv8 a53)
+     *          3 current core is big core with wider load (e.g. ARMv8 a72)
+     */
+    MlasCoreType GetCoreType()
```

Comment:
this will cause multi-definition link error.

Reply:
I don't see the error when compile this. how to reproduce?

Reply:
This is defined inside the class, should be considered inline by the compiler. I also added an artificial reference in another cpp file and it compiled fine. If you know how to produce the linking error please let me know.

Reply:
don't see it is under MlasPlatform. I thought it is a standalone function.